ui/src/app/cohort-search/cohort-builder/cohort-builder.component.ts

```diff
@@ -1,63 +1,60 @@
 import {
   Component,
-  ChangeDetectionStrategy,
-  ChangeDetectorRef,
-  Inject,
   OnInit,
   OnDestroy,
 } from '@angular/core';
-import {DOCUMENT} from '@angular/platform-browser';
 import {NgRedux, select} from '@angular-redux/store';
 import {Router, ActivatedRoute} from '@angular/router';
 import {Observable} from 'rxjs/Observable';
 import {Subscription} from 'rxjs/Subscription';
+import {List} from 'immutable';
 
-import {CohortSearchActions} from '../redux/actions';
 import {
+  CohortSearchActions,
   CohortSearchState,
-  inclusionGroups,
-  exclusionGroups,
+  includeGroups,
+  excludeGroups,
   wizardOpen,
-  subjects,
-} from '../redux/store';
+  totalCount
+} from '../redux';
 
 @Component({
   selector: 'app-cohort-builder',
   templateUrl: './cohort-builder.component.html',
   styleUrls: ['./cohort-builder.component.css'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  // changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
I'm assuming that were keeping this around because we will need it to correct the scrollable div after adding a new search group?